backend/protocol_rpc/endpoint_generator.py

```diff
@@ -1,24 +1,54 @@
 # rpc/endpoint_generator.py
-
+import traceback
 from functools import partial, wraps
 from typing import Callable
 from flask_jsonrpc import JSONRPC
 from backend.protocol_rpc.message_handler.base import MessageHandler
+from backend.protocol_rpc.types import EndpointResult, EndpointResultStatus
+
+"""
+Add the function name to this list to disable info logs for that endpoint
+The info logs are the Starting... and Endpoint successfully executed messages
+"""
+disableInfoLogs = [
+    "get_transaction_by_id",
+    "get_contract_schema_for_code",
+    "get_contract_schema",
+]
```

Comment:
nit: it'd be great if we could configure this through envvars. We can encode this as a JSON in an envvar and parse it here

Reply:
Done!